pkg/repo/chartrepo.go

```diff
@@ -197,28 +197,29 @@ func (r *ChartRepository) generateIndex() error {
 
 // FindChartInRepoURL finds chart in chart repository pointed by repoURL
 // without adding repo to repositories
-func FindChartInRepoURL(repoURL, chartName, chartVersion, certFile, keyFile, caFile string, getters getter.Providers) (string, error) {
-	return FindChartInAuthRepoURL(repoURL, "", "", chartName, chartVersion, certFile, keyFile, caFile, getters)
+func FindChartInRepoURL(repoURL, chartName, chartVersion, certFile, keyFile, caFile string, insecureSkipTLSverify bool, getters getter.Providers) (string, error) {
```

Comment:
as this changes  a public function signature, this would be considered a breaking change according to our contract, and therefore we cannot accept this PR as it currently stands.

Can you please make the necessary modifications to keep the function signatures intact. Have a look at how `FindChartInAuthRepoURL` was first introduced.

Given the size of this function's signature it may be worth looking into a code refactor.

Reply:
sorry， I will update it. thank you very much!

Reply:
According to your suggestion, I change the code,  I don't know whether the modification meets the requirements. Thank you very much, I love helm.